src/components/home/HomePage.js

```diff
@@ -1,33 +1,75 @@
+/* eslint-disable prettier/prettier */
```

Comment:
I think this line will not be approved by anyone. 👀

Reply:
okk sir. I will remove it. xD